scripts/convert_algoritmeregister.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
Shouldn't this script call the conversion util?

Reply:
For now I followed Anne's suggestion to chain everything in the shell script.